api/review_latency_api_test.py

```diff
@@ -60,7 +60,7 @@ def tearDown(self):
 
   def test_do_get__nothing_requested(self):
     """When no reviews have been started, the result is empty."""
-    actual = self.handler.do_get()
+    actual = self.handler.do_get(today=self.today)
```

Comment:
Isn't self.today is hardcoded to datetime(2024, 3, 22)? How will this resolve the out-of-date timestamp issue

Reply:
Yes, self.today is hard coded, and the dates of the review requests are also hard coded, so they will be within 90 days even as we continue to run the same test months or years from now.     

The problem was that when I was not passing in a hard-coded value for self.today, then it was defaulting to the current datetime.datetime.now() and that was getting farther and farther away from the hard-coded review request date, so the test started failing once it was more than 90 days.

Reply:
Aha I see. I think normally we can mock the clock if self.today cannot be hardcoded, but there is no need for it here.